integration/vite-route-added-test.ts

```diff
@@ -38,7 +38,12 @@ test.describe(async () => {
   });
   test.afterAll(() => stop());
 
-  test("Vite / dev / route added", async ({ page }) => {
+  test("Vite / dev / route added", async ({ page, browserName }) => {
+    test.skip(
+      browserName === "webkit",
+      "Safari caches too aggressively, browser manifest is cached with old routes"
+    );
+
```

Comment:
We also needed to do this in React Router, seems to be triggered by the Playwright update: https://github.com/remix-run/react-router/pull/11511